packages/formation/sass/modules/_m-loading-indicator.scss

```diff
@@ -2,10 +2,22 @@
   text-align: center;
 
   .loading-indicator {
-    min-height: 8em;
-    background: url('#{$formation-image-path}/loading-state.svg') center center no-repeat;
-    background-size: contain;
-    text-align: center;
+    display: block;
+    height: 8em;
+    margin-top: 2em;
+
+    &:after {
+      content: " ";
+      display: inline-block;
+      width: 7em;
+      height: 7em;
+      margin: 1px;
+      border-radius: 50%;
+      border: 8px solid #0071BB;
```

Comment:
Let's use our color tokens instead of hex codes here: `$color-primary` is the token for `#0071BB`.

Reply:
🤦‍♂ I must have been really tired when I wrote this!